src/extensions/bigbang/banner.go

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package bigbang contains the logic for installing Big Bang and Flux
+package bigbang
+
+import (
+	"github.com/pterm/pterm"
+)
+
+func printBanner() {
```

Comment:
This doesn't go anywhere...

Reply:
NVM I see I had a too small terminal.  Can we make a normal header in that case?

                                                                                       
  📦 BIGBANG COMPONENT

Reply:
Also also, we should respect `--no-progress`